### ISSUE TITLE
Path variables in graphql mapping

### ISSUE THREAD

**Issue by albercevam** [graphql-java-kickstart/graphql-spring-boot#246]
I have a use case where I need to have a dynamic path with path variables like `/projects/api/platform/{platformId}/graphql` where `platformId` is gathered at runtime. 
In previous versions, I could use a `SimpleGraphqlHttpServlet` to expose the Rest endpoint like : 
```kotlin
@RequestMapping("`/projects/api/platform/{platformId}/graphql")
    @Throws(Exception::class)
    fun graphql(@PathVariable platformId: String, request: HttpServletRequest, response: HttpServletResponse) {
        graphQLServlet.service(request, response)
    }
```
Now, with `GraphQLHttpServlet` I have an HTTP 500 error without stackTrace.
 
I saw that `graphiql-spring-boot-starter` accept path variables in his mapping in `application.yml` like :
```yaml
graphiql:
  mapping: /projects/api/platform/{platformId}/graphiql
  endpoint:
    graphql: /projects/api/platform/{platformId}/graphql
    subscriptions: /projects/api/platform/1/subscriptions
```
So, I wonder if it is possible to have the same behavior for graphql mapping

I also noticed that for graphiql configuration in application.yml, you can add path variables for query/mutation but not for subscriptions.

**Comment by oliemansm** [comment]
GraphiQL is served using a Controller, while GraphQL itself is served using a `ServletRegistrationBean` which doesn't appear to easily support path variables. I've found the culprit though. The backend was logging a stack trace in my case which clearly identified the problem in `graphql-java-servlet`. Will fix it in 5.9.1.

**Comment by armkillbill** [comment]
How to do this in 2023? There is no document or example about this.
